README.md

```diff
@@ -68,45 +68,31 @@ will output an image logo url in the format `https://my.image.service/foo/v1/ima
 
 #### Silent mode ([docs](http://origami.ft.com/docs/syntax/scss/#silent-styles))
 
-When using `o-header` in silent mode, we offer a series of helper mixins to output styles for different parts of the header. The header is made up of various features (e.g. `Nav` and `Search`), has different configurations like `sticky` and `simple` and can be combined with additional features, such as the `drawer` and `megamenu`.
+When using `o-header` in silent mode, we offer a series of helper mixins to output styles for different parts of the header.
 
 ##### oHeader
 
-To get the default header, use the `oHeader` mixin. The `oHeader` mixin will output the base styles for the header and accepts a list of the feature styles it should also output. By default the mixin will output all the feature styles if no list is specified.
+The header is made up of various features (e.g. `Nav`, `Search`, and `Drawer`). To get everything, use the `oHeader()` mixin without arguments. To get only the stuff you need, you can pass in a list of features.
```

Comment:
here, you've capitalised `Nav` etc, but below they aren't capitalised. Should the user be passing in capitalised arguments or not?